### ISSUE TITLE
HTML and CSS are not linked together at all - Score NA

### ISSUE THREAD

**Issue by darcboykin1** [emjogall/Dasmodo#2]
Looks like you were missing a folder when pushing your work to github. As you can see here, the link that you're using in the head of your HTML is calling a directory that doesn't exist:

![Screen Shot 2019-03-25 at 11 47 24 PM](https://user-images.githubusercontent.com/23035535/54970451-82b30f00-4f58-11e9-859d-165a891e340a.png)

**Comment by emjogall** [comment]
Thanks for that - so I needed to add the resources folder as well. I'm guessing I also needed to add the pictures into that file too?

**Comment by darcboykin1** [comment]
> Thanks for that - so I needed to add the resources folder as well. I'm guessing I also needed to add the pictures into that file too?

Yeah, you can do that. If you do, just make sure you change file paths around to reflect that
